content/nginxaas-google/overview.md

```diff
@@ -42,22 +42,23 @@ The key capabilities of NGINXaaS for Google Cloud are:
 
 NGINXaaS for Google has a global presence with management requests being served from various geographical controllers. A Geographical Controller (GC) is a control plane that serves users in a given geographical boundary while taking into account concerns relating to data residency and localization. Example: A US geographical controller serves US customers. We currently have presence in two Geographies: **US** and **EU**.
 
+## Supported regions
+
+{{< include "/nginxaas-google/supported-regions.md" >}}
+
 ### Networking
 
 We use Google [Private Service Connect]((https://cloud.google.com/vpc/docs/private-service-connect)) (PSC) to securely connect NGINXaaS to your applications and enable client access to your deployments. A [PSC backend](https://cloud.google.com/vpc/docs/private-service-connect#backends) brings the NGINXaaS deployment into your client network, allowing your application clients to connect seamlessly. A [PSC Interface](https://cloud.google.com/vpc/docs/private-service-connect#interfaces) brings the deployment into your application network, enabling secure connectivity to your applications. This approach gives you full control over traffic flow by leveraging your own networking resources, so you can apply your preferred security controls and ensure a secure deployment environment.
 
 
-## Supported regions
-
-{{< include "/nginxaas-google/supported-regions.md" >}}
+## Current Limitations
 
-## Limitations
+At present, there are some constraints to be aware of regarding NGINXaaS for Google Cloud. We are actively working to enhance the offering, and many of these constraints may evolve over time as we expand and enhance our service.
```

Comment:
I like the changes you made but usually just listing these limitations with no further guidance to users is also not as good of an experience.

Maybe indicate that in case a user is interested in any of these features, that they should open up a support request with us? or reach out to us in a certain way so we know.